packages/search-in-workspace/src/node/ripgrep-search-in-workspace-server.slow-spec.ts

```diff
@@ -25,14 +25,17 @@ import { ILogger, isWindows } from '@theia/core';
 import { MockLogger } from '@theia/core/lib/common/test/mock-logger';
 import { RawProcessFactory, RawProcessOptions, RawProcess, ProcessManager } from '@theia/process/lib/node';
 import * as path from 'path';
-import { FileUri } from '@theia/core/lib/node/file-uri';
+import { FileUri } from '@theia/core/src/node';
```

Comment:
Please don't use import from the `src` directory (I know that auto-import does that, but it's wrong). Actually, you don't need to change this line I believe.

Reply:
You are absolutely correct.  It would have been removed and re-added by the tooling and I didn't notice.